google-apis-common/src/lib.rs

```diff
@@ -333,6 +333,13 @@ pub struct MultiPartReader<'a> {
 }
 
 impl<'a> MultiPartReader<'a> {
+    // TODO: This should be an associated constant
+    /// Returns the mime-type representing our multi-part message.
+    /// Use it with the ContentType header.
+    pub fn mime_type() -> Mime {
+        Mime::from_str(&format!("multipart/related;boundary={}", BOUNDARY)).expect("valid mimetype")
```

Comment:
It's a bit strange that this can't be done statically. Does an API exist to build any mime type, or is string parsing for convenience?
Ultimately I don't mind as I don't think it has any impact on performance, maybe it's more readable even than the related API usage (if it exists).

Reply:
I also thought the same, but `mime` isn't being actively maintained, and lacks quite a few features, including an easy way to do compile time `mime`.